automatic/jenkins/tools/LICENSE.txt

```diff
@@ -0,0 +1,23 @@
+From: https://github.com/jenkinsci/jenkins/blob/master/LICENSE.txt
```

Comment:
We don't follow the convention of using `From:` in the license file in this repository.

That should be part of the `VERIFICATION.txt` file instead.
https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/7zip.install/legal/VERIFICATION.txt#L19

The license should only have the actually license content, nothing else. And it should be kept in the `legals` folder.

https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md#114-embed-a-package-if-allowed